x-pack/plugin/fleet/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/fleet/global_checkpoints.yml

```diff
@@ -0,0 +1,50 @@
+setup:
+  - do:
+      indices.create:
+        index: test-2-shards
+        body:
+          settings:
+            number_of_shards: "2"
+
+  - do:
+      indices.create:
+        index: test-1-shard
+        body:
+          settings:
+            number_of_shards: "1"
+
+  - do:
+      index:
+        index:  test-1-shard
+        body:   { max: 2 }
+
+  - do:
+      indices.refresh: {}
+
+---
+"Get Global Checkpoints":
+  - do:
+      fleet.global_checkpoints:
+        index: "test-2-shards"
+
+  - match: { global_checkpoints.0: -1 }
+  - match: { global_checkpoints.1: -1 }
+
+---
+"Get Global Checkpoints after advance":
```

Comment:
I think we can also add a test using `wait_for_advance=true` that does not timeout?

Reply:
I did not find this test in this .yml, perhaps you added it elsewhere?